frontend-runitup/src/App.jsx

```diff
@@ -1,12 +1,13 @@
 import './App.css'
 import SignUp from './Components/Signup_Page/SignUp'
-
+import LoginPage from './Components/Login_Page/LoginPage'
```

Comment:
Small nit but you could consider rethinking how you structure your code. It might be a lot to create a separate folder for each page. For ex: have folders like
- components
- styles
- utils
- etc

Then within styles, you add styling css files. And within components, you can add subfolders. Example could be: authentication (which would include everything that has to do with this: login page for ex), layout, feed, routes etc. Utils folder could contain all of the utility files/functions that are reused across different components.